api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/calculation/BasePatientCalculationTest.java

```diff
@@ -0,0 +1,135 @@
+package org.openmrs.module.eptsreports.reporting.intergrated.calculation;
+
+import java.util.Collection;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.api.context.Context;
+import org.openmrs.calculation.patient.PatientCalculation;
+import org.openmrs.calculation.patient.PatientCalculationContext;
+import org.openmrs.calculation.patient.PatientCalculationService;
+import org.openmrs.calculation.result.CalculationResult;
+import org.openmrs.calculation.result.CalculationResultMap;
+import org.openmrs.module.eptsreports.api.EptsReportsService;
```

Comment:
this as well, probably u just didn't branch off the test framework branch, try doing that so save u the pain of having to reset and throw away all these